code/src/reducers/todos.js

```diff
@@ -0,0 +1,46 @@
+import { createSlice } from '@reduxjs/toolkit'
+
+const initialState = {
+  //det vi behöver för att göra appen, alltså en lista på sakerna
+  //hur det ska se ut det vi kör in i storen
+  list: {
+    //namnet på listan
+
+    //innehållet, aLLtså tasksen
+    items: [
+      // varje task är ett object som har en description och en boolean
+
+    ]
+  }
+}
+
+export const todos = createSlice({
+  name: 'todos',
+
+  initialState: initialState,
+  reducers: {
+    //FATTAR INTE DETTA State är current state, att den tar den uppdaterade.
+    //action will ta med sig det vi får in från dispatchen
+    addTodo: (state, action) => {
+      const { itemInfo } = action.payload
+      //detta uppdaterar staten med det vi kört in. letar upp list i staten, sen listan, sen i items sätter den in ett nytt obejkt
+      state.list.items.push(itemInfo)
+    },
```

Comment:
Add a unique id to your todos that you can use instead of the index, eg. a timestamp, to be able to control your input better. In Jennie's codealong from week 15 you can get a hint on how: [https://github.com/Technigo/codealong-week15-20april/blob/master/src/reducers/fridge.js](https://github.com/Technigo/codealong-week15-20april/blob/master/src/reducers/fridge.js)